### ISSUE TITLE
When the batch size is greater than 5, it will cause insufficient memory. Here are some potential solutions

### ISSUE THREAD

**Issue by xiao-ze-123** [bytedance/LatentSync#138]
If you just want to increase batch size. There are many methods:
1. Implement Gradient checkpointing
2. Gradient accumulation
3. Implement DeepSpeed
4. More GPUs

What batch size does everyone use during training, and how many total steps do you run?

**Comment by chunyu-li** [comment]
You can refer to the issue https://github.com/bytedance/LatentSync/issues/3
